ceph-salt-formula/salt/ceph-salt/files/registries.conf.j2

```diff
@@ -3,7 +3,11 @@
 # For more information on this configuration file, see containers-registries.conf(5)
 
 # An array of host[:port] registries to try when pulling an unqualified image, in order
+{% if grains['os_family'] == 'Suse' %}
+unqualified-search-registries = ["registry.suse.com"]
```

Comment:
what's `grains['os_family']` for opensuse?

Reply:
It's `Suse`:

```
master:~ # cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="15.2 Beta"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.2"
PRETTY_NAME="openSUSE Leap 15.2 Beta"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.2"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"

master:~ # salt-call grains.get os_family
local:
    Suse
```

Reply:
which would be wrong to point leap to registry.suse.com. hm. I guess we need to distinguish opensuse and sle here.

Reply:
We could deal with that later, in a follow-up PR. This patch solves the immediate problem. (After all, this code path is not even triggered in the typical use case, right?)

Reply:
> which would be wrong to point leap to registry.suse.com. hm. I guess we need to distinguish opensuse and sle here.

We could check the "os_family" first, and if it is "Suse", we could then look into the "osfinger" grain to distinguish between SLE and openSUSE.

Reply:
@sebastian-philipp where should we point `Leap`?

Reply:
I've now updates this PR to also check `osfullname`.

Leap:
```
# salt-call grains.get os_family
local:
    Suse

# salt-call grains.get osfullname
local:
    Leap
```

SLES:
```
# salt-call grains.get os_family
local:
    Suse

# salt-call grains.get osfullname
local:
    SLES
```